src/components/products/products.tsx

```diff
@@ -0,0 +1,48 @@
+import { FC, useEffect } from "react";
+
+import ProdPC from "../prodPC/prodPC";
+import ProdPS from "../prodPS/prodPS";
+import ProdXbox from "../prodXbox/prodXbox";
+
+interface ProductsProps {
+  iMadeError: boolean;
+  currentChoice: string;
+  children?: React.ReactChild | React.ReactNode;
+}
+
+const Products: FC<ProductsProps> = ({ currentChoice, iMadeError }) => {
+  if (iMadeError) {
+    throw new Error("Smth went wrong");
+  }
+
+  // useEffect(() => {}, [currentChoice]);
+
+  const links = {
+    home: "/",
+    products: "/products",
+    about: "/about",
+    PC: "/PC",
+    PS: "/PS",
+    Xbox: "/Xbox",
+  };
+
+  return (
+    <div>
+      <div>Products page</div>
+      {currentChoice === "" ? (
+        <div>
+          <ProdPC />
```

Comment:
Do you need to create separate components just to hold a div? At least you can create common component to store this logic